README.md

```diff
@@ -1,2 +1,219 @@
 # Stator
+
 Stator, your go-to template for the perfect stack.
+
+## Why?
+
+Have you ever started a new project by yourself?
+If so, you probably know that it is tedious to set up all the necessary tools.
+Just like you, the part I enjoy the most is coding, not boilerplate.
+stator solves all of this for you by using the latest technologies' most up-to-date standards.
+Indeed, this template is opinionated as to what's the best, but it does enforce excellent practices such as **code re-usability**, **enforces coding guidelines**, **usage of a monorepo**, etc.
+
+If you want more details about how this idea was implemented, I recommend reading the [series of blog articles](https://yann510.hashnode.dev/creating-the-modern-developer-stack-template-part-1-ckfl56axy02e85ds18pa26a6z) I wrote on the topic.
+
+### Demo Application
+
+![demo application](readmeAssets/todo-demo.gif)
+
+## Getting started
+
+### Introduction
+
+This template includes a demo todo application that serves as an example of sound patterns.
+Of course, you won't be creating a todo application for your project, but you can use this as an example of useful patterns and learn how to use the technologies presented in this project.
+
+### Prerequisites
+
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- [node.js](https://nodejs.org/en/download/) your version must >= 14
+
+This repository is a repository template, which means you can use the `Use this template` button at the top to create your project based on this.
+
+![use template button](readmeAssets/useTemplate.png)
+
+\*Note: If you have an existing repository, this will require more work. I would recommend using the `use template button` and migrating your current code to the newly created projects.
+
+You will now want to make this project yours by replacing all `stator` occurrences with your own project name.
+Thankfully, we have a script just for that:
+
+`node renameProject.js --project-name {YOUR_PROJECT_NAME}`
+
+\*Note: I highly recommend that the project name is the same as your git repository.
+
+The file will delete itself once it has been completed.
+On completion, you will see the following message:
+
+![project appropriation success](readmeAssets/projectAppropriationSuccess.png)
+
+You can now install the dependencies:
+
+`npm i`
+
+Running the database:
+
+`npm run postgres`
+
+Running the api:
+
+`nx serve api`
```

Comment:
maybe tell them to use npm start api instead?

Reply:
since you're saying all commands are in package.json

Reply:
Good point.